src/pybind/mgr/orchestrator_cli/module.py

```diff
@@ -268,6 +268,16 @@ def _rgw_rm(self, svc_id):
     def _nfs_rm(self, svc_id):
         return self._rm_stateless_svc("nfs", svc_id)
 
+    @CLIWriteCommand('orchestrator nfs update',
+                     "name=svc_id,type=CephString "
+                     "name=num,type=CephInt",
+                     'Scale an NFS service')
+    @handle_exceptions
+    def _nfs_update(self, svc_id, num):
```

Comment:
could you add a test that tests this command in `/qa/tasks/mgr/test_orchestrator_cli.py`?